compass-js-frontend/src/main/webapp/ui/page/editor.js

```diff
@@ -17,20 +18,40 @@ var EditorPage = BasePage.extend({
 	pageTitle: 'Editor',
 	template: template,
 	scenario: undefined,
+	root: undefined,
 	initialize: function (scenarioId, options) {
 		//The router gives as a string, but the model wants a number...
 		var idAsNumber = parseInt(scenarioId);
 		this.scenario = new Scenario({id: idAsNumber});
-		this.scenario.fetch({
-			success: this.initUI.bind(this)
-		});
+		this.fetchData();
 	},
 	render: function () {
 		this.renderWithTemplate();
 		return this;
 	},
-	initUI: function () {
+	initUI: function (rootNode) {
+		this.root = rootNode;
 		console.log("Editor fetched scenario: " + this.scenario.name);
+		console.log("Editor fetched root node: " + this.root.name);
```

Comment:
Do we really want these logs here? Maybe trigger an event that signals loading is done? (which in turn can propery initialize UI, if needed?)